util.py

```diff
@@ -78,6 +81,35 @@
     report = report.format(message.author.mention, sanitize(username))
     await client.send_message(message.channel, report)
 
+async def command_unset_username(client, message):
+    command = 'unset'
+    content = message.content.lower().strip()
+    if content[1:] != command: return
+
+    if not message.author.id in client.usernames:
+        report = ':bangbang: You haven\'t set a username yet. Use the `set` command to do so'
+        await client.send_message(message.channel, report)
+        return
+    
+    del client[message.author.id]
+
+    headers = { 'Content-Type': 'application/json; charset=utf-8', 'Data-Type': 'json', }
+    r = requests.put(client.bin, data=json.dumps(client.usernames), headers=headers)
+
+    if r is None:
+        report = ':no_entry: Failed to make HTTP request'
+        await client.send_message(message.channel, report)
+        return
+
+    if r.status_code != requests.codes.ok:
+        report = ':no_entry: Failed to set username. Error code: **{}**'.format(r.status_code)
+        await client.send_message(message.channel, report)
+        return
+
+    report = ':white_check_mark: {} your last.fm username has been unset'
+    report = report.format(message.author.mention)
+    await client.send_message(message.channel, report)
+
```

Comment:
You also need to add a call to `command_unset_username` in `self.on_message` in `main.py`